src/enr/crypto.ts

```diff
@@ -0,0 +1,63 @@
+// @ts-ignore
+import keccak from "keccakjs";
```

Comment:
It would probably be nicer to declare module type like
```javascript
//src/types/keccakjs/index.d.ts
declare module "keccakjs" {
   export default class Keccak...
}
```